backend/src/main/java/com/example/backend/controller/NFTCollectionsController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.backend.controller;
+
+import com.example.backend.model.MetadataFromCollection;
+import com.example.backend.model.NFTCollectionsResponseElement;
+import com.example.backend.service.CollectionsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api.nftport.xyz/v0/")
```

Comment:
Ich würde den Request umbenennen damit er unabhängig von NFT Port ist.


```suggestion
@RequestMapping("/api/nft")
```